test/env.js

```diff
@@ -3,8 +3,134 @@ process.env.METAMASK_ENV = 'test';
 /**
  * Used for testing components that use the Icon component
  * 'ui/components/component-library/icon/icon.js'
+ * There is a high possibility that this list becomes
+ * out of sync with production because it will need to be
+ * manually updated every time there is an icon added or removed
+ *
+ * TODO: Automate the creation of ICON_NAMES env var using the generateIconNames script in ../development/generate-icon-names
```

Comment:
Any opinions on whether this is worth automating @Gudahtt or whether it would be sufficient to add a step to the add icon docs?

Reply:
Hmm. It's already automated by the `generateIconNames` function, we should be able to utilize that. I guess the problem is that that function is async.

Could we use top-level await here? I don't recall if that works yet with our unit test setup.

Reply:
Thank you for the help 🙏 have updated the PR